addons/position_manager/position_manager.lua

```diff
@@ -43,22 +43,28 @@ config = require('config')
 default = {
     x = 0,
     y = 0,
+    delay = 0,
 }
 
 settings = config.load(default)
 
 function move(settings)
+    if settings.delay > 0 then
+        coroutine.sleep(settings.delay)
+    end
     windower.send_command('wincontrol move %s %s':format(settings.x, settings.y))
 end
 
-function handle_commands(cmd, pos_x, pos_y, name)
-    cmd = cmd:lower() or 'help'
+function handle_commands(cmd, ...)
+    cmd = cmd:lower()
 
     if cmd == 'r' then
         windower.send_command('lua r position_manager')
-    elseif cmd == 'set' and pos_x and pos_y then
+        return
+    elseif cmd == 'set' then
+        local name = arg[3]
```

Comment:
Are you sure that works? I know they removed that feature from Lua (finally), but not sure in what version, I did think it was 5.1. Though I guess you tested it?

I still recommend explicitly defining a table, makes it appear less magicky, and is more in line with modern Lua.

Reply:
It does work correctly in Windower, but I agree with your last statement. Fixed.